games-arcade/circuslinux/circuslinux-1.0.3-r1.ebuild

```diff
@@ -1,37 +1,36 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit desktop
+EAPI=8
+
+inherit desktop toolchain-funcs
 
 DESCRIPTION="clone of the Atari 2600 game \"Circus Atari\""
-SRC_URI="ftp://ftp.sonic.net/pub/users/nbs/unix/x/circus-linux/${P}.tar.gz"
+SRC_URI="ftp://ftp.tuxpaint.org/unix/x/circus-linux/src/${P}.tar.gz"
 HOMEPAGE="http://www.newbreedsoftware.com/circus-linux/"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~x86"
-IUSE=""
 
-DEPEND="media-libs/libsdl
+DEPEND="
+	media-libs/libsdl[sound,video]
```

Comment:
Need revbump for this change and, if adding --enable-joystick, obviously joystick :)